src/subscript/genertobs_unstable/_datatypes.py

```diff
@@ -0,0 +1,325 @@
+"""Pydantic models for genertobs"""
+
+import logging
+import warnings
+from enum import Enum
+from pathlib import Path
+from typing import Dict, List, Union
+
+from pydantic import (
+    BaseModel,
+    ConfigDict,
+    Field,
+    RootModel,
+    computed_field,
+    field_validator,
+    model_validator,
+)
+
+
+def is_number(tocheck):
+    """Check that variable can be converted to number
+
+    Args:
+        tocheck (something): what shall be checked
+
+    Returns:
+        bool: check passed or not
+    """
+    try:
+        float(tocheck)
+        return True
+    except TypeError:
+        return False
+
+
+def is_percent_range(string):
```

Comment:
This doesn't seem to check if the string contains %

Reply:
But I see that the checking is done before calling the function. Could probably update the docstring.